_posts/2022-11-21-november-demo-day.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: Dask Demo Day November 2022
+author: Richard Pelgrim (Coiled)
+tagline: an optional tagline
+tags: [Community]
+theme: twitter
+draft: true
+---
+
+{% include JB/setup %}
+
+# Dask Demo Day : November
+
+Once a month, the Dask Community team hosts Dask Demo Day: an informal and fun online hangout where folks can showcase new or lesser-known Dask features…and the rest of us can learn about all the things we didn’t know Dask could do 😁
+
+November’s Dask Demo Day had 5 great demo's. We learned about:
+- the Dask-Optuna integration for distributed hyperparameter optimization
+- visualizing 2-billion lightning flashes with Dask, RAPIDS and Datashader
+- the new Dask CLI
+- Dask-Awkward
+- profiling your Dask code with Dask-PySpy 
+
+This blog gives you a quick overview of the 5 demos and demonstrates how they might be useful to you. You can watch the full recording below.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/_x7oaSEJDjA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Visualization at Lightning Speed
+
+Would it be possible to interactively visualize all the lightning strikes in his dataset, Kevin Tyle (University of Albany) recently asked himself. In this demo, Kevin shows you how he leveraged CUDA, RAPIDS-AI, Dask and Datashader to build a smooth interactive visualization of 8 years’ worth of lightning strikes. That’s over 2 billion rows of data.
```

Comment:
Link to Kevin somehow (Twitter, UofA staff page, etc).

Links to all the projects mentioned.

Reply:
Done